src/Todo.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import './Todo.css';
+
+interface TodoItem {
+  todo: string;
+  completed: boolean;
+}
+
+const Todo = () => {
+  const [todo, setTodo] = useState<string>('');
+  const [array, setArray] = useState<TodoItem[]>([]);
+
+  const addTodo = () => {
+    if (todo.trim() !== '') {
+      setArray([...array, { todo, completed: false }]);
+      setTodo('');
+    } else {
+      alert('Enter task');
+    }
+  };
+  const handleInput = (str: string) => {
+    setTodo(str);
+  };
+
+  const deleteTodo = (index: number) => {
+    const newArray = [...array];
+    newArray.splice(index, 1);
+    setArray(newArray);
+  };
+
+  const Completion = (index: number) => {
```

Comment:
Always remember in react we don't name the handler functions or helper functions with initial capital letters.
Name with initial letter capital is always used for Functional Components.
& for handler functions like this one you can name it as
onCompletion
handleCompletion ...etc ie. following snake case